easybuild/easyconfigs/h/hanythingondemand/hanythingondemand-2.1.4-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'hanythingondemand'
+version = '2.1.4'
+
+homepage = 'https://github.com/hpcugent/hanythingondemand'
+description = """HanythingOnDemand (HOD) is a system for provisioning virtual Hadoop clusters over a large physical cluster.
+It uses the Torque resource manager to do node allocation."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = [
+    'https://github.com/hpcugent/hanythingondemand/archive/'
+]
+sources = ['%s.tar.gz' % version]
+
+python = 'Python'
+pythonver = '2.7.9'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % (python, pythonver)
+
+installopts='--single-version-externally-managed --record=record'
+
+# a Python version with mpi4py and pbs_python (vsc.fancylogger) is required at runtime
+dependencies = [
+    (python, pythonver),
+    ('pbs_python', '4.3.5', versionsuffix),
+    ('vsc-mympirun', '3.4.2', '', True),
+]
+
+options = {'modulename': 'hod'}
```

Comment:
no `modulename`, i.e. the name used for the `python -c 'import <modname>'` sanity check, this is correct